ee-9/feature-pack/src/main/resources/modules/system/layers/base/org/hibernate/commons-annotations/main/module.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2012, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<!-- Represents the Hibernate 4.0.x module-->
+<module xmlns="urn:jboss:module:1.9" name="org.hibernate.commons-annotations">
+    <resources>
+        <artifact name="${org.hibernate.common:hibernate-commons-annotations}"/>
+    </resources>
+
+    <dependencies>
+       <module name="org.jboss.logging"/>
+    </dependencies>
+</module>
```

Comment:
This appears to just duplicate the one used by standard WildFly, so if that continues to be the case as your work on this continues it should be removed.

Reply:
removing...